js/main.js

```diff
@@ -1,33 +1,106 @@
 /* global data */
 /* exported data */
-var entryForm = document.querySelector('.form');
-var photo = document.querySelector('img');
-var photoUrl = document.querySelector('.photoUrl');
-var title = document.querySelector('.title');
-var notes = document.querySelector('.notes');
-
-photoUrl.addEventListener('input', updatePhoto);
+var $entryForm = document.querySelector('.form');
+var $photo = document.querySelector('img');
+var $photoUrl = document.querySelector('.photoUrl');
+var $title = document.querySelector('.title');
+var $notes = document.querySelector('.notes');
+var $ul = document.querySelector('ul');
+var $viewElements = document.querySelectorAll('.view');
+var $entriesEmpty = document.querySelector('.entries-empty');
+var i;
 
+$photoUrl.addEventListener('input', updatePhoto);
 function updatePhoto(event) {
-  photo.setAttribute('src', photoUrl.value);
+  $photo.setAttribute('src', $photoUrl.value);
 }
 
-entryForm.addEventListener('submit', saveEntryValues);
-
+$entryForm.addEventListener('submit', saveEntryValues);
 function saveEntryValues(event) {
   event.preventDefault();
 
   var newEntry = {
     entryId: data.nextEntryId,
-    photoUrl: photoUrl.value,
-    title: title.value,
-    notes: notes.value
+    photoUrl: $photoUrl.value,
+    title: $title.value,
+    notes: $notes.value
   };
 
   data.nextEntryId++;
   data.entries.unshift(newEntry);
-  photo.setAttribute('src', 'images/placeholder-image-square.jpg');
-  photoUrl.value = '';
-  title.value = '';
-  notes.value = '';
+  $photo.setAttribute('src', 'images/placeholder-image-square.jpg');
+  $photoUrl.value = '';
+  $title.value = '';
+  $notes.value = '';
+
+  for (i = 0; i < $viewElements.length; i++) {
+    if ($viewElements[i].getAttribute('data-view') !== 'entries') {
+      $viewElements[i].classList.add('hidden');
+    } else $viewElements[i].classList.remove('hidden');
+  }
+  data.view = 'entries';
+  location.reload();
```

Comment:
Oh nooooooo!!! `location.reload` is a tool you should always avoid, it completely reloads the page. This is essentially the same as deciding that you want your couch on the other side of your living room, but instead of just moving the couch, you blow up your house and build a new one with the couch where you want 😂 

What is the reason you are doing this? It is likely you just need to reset the value of a variable or two, which you can do yourself in your code.